src/app/components/clt-editor/hints/hint-list.component.ts

```diff
@@ -1,7 +1,10 @@
-import {Component, Input, ChangeDetectionStrategy} from "@angular/core";
+import {Component, Input, Output, ChangeDetectionStrategy} from "@angular/core";
 import {ComponentBase} from "../../common/component-base";
 import {ExternalLinks} from "../../../cwl/external-links";
 import {ExpressionModel} from "cwlts/models/d2sb";
+import {FormControl} from "@angular/forms";
+import {ReplaySubject} from "rxjs";
+import {RequirementBaseModel} from "cwlts";
```

Comment:
Import from "cwlts/models/d2sb" otherwise webpack will not be able to resolve the dependency.